packages/mask/src/plugins/Trader/trader/useGetTradeContext.ts

```diff
@@ -175,6 +179,30 @@ export function useGetTradeContext(tradeProvider?: TradeProvider) {
                     ADDITIONAL_TOKENS: {},
                     CUSTOM_TOKENS: MDEX_CUSTOM_BASES,
                 }
+            case TradeProvider.VENONSWAP:
+                return {
+                    TYPE: tradeProvider,
+                    IS_UNISWAP_V2_LIKE: true,
+                    GRAPH_API: DEX_TRADE.VENONSWAP_THEGRAPH,
+                    INIT_CODE_HASH: DEX_TRADE.VENONSWAP_INIT_CODE_HASH,
+                    ROUTER_CONTRACT_ADDRESS: DEX_TRADE.VENONSWAP_ROUTER_ADDRESS,
+                    FACTORY_CONTRACT_ADDRESS: DEX_TRADE.VENONSWAP_FACTORY_ADDRESS,
```

Comment:
Did you forget to add `SPENDER_CONTRACT_ADDRESS`?

Reply:
yes, add spender contract address on last commit